zaza/openstack/charm_tests/cinder/tests.py

```diff
@@ -153,7 +153,7 @@ def test_120_volume_force_delete(self):
 
     @property
     def services(self):
-        """Return a list services for OpenStack release."""
+        """Return a list services for the selected Openstack release."""
```

Comment:
According to https://www.openstack.org/ it really is "OpenStack".  Perhaps a global search/replace to pick them all up in comments?